content/300-accelerate/600-faq.mdx

```diff
@@ -182,3 +182,17 @@ As the cache needs to be cleared globally, it is difficult to provide a specific
 **Invalidate**: The cache entry is deleted, and new data will be fetched on the next request, causing a cache miss. This removes stale data but may lead to slower responses until the cache is repopulated.
 
 **Revalidate**: The cache entry is updated proactively, ensuring the next request uses fresh data from the cache. This keeps the cache valid and maintains faster response times by avoiding cache misses.
+
+### What is on-demand cache invalidation?
+
+[On-demand cache invalidation](/accelerate/caching#on-demand-cache-invalidation) lets applications instantly update specific cached data when it changes, instead of waiting for regular cache refresh cycles. This keeps information accurate and up-to-date for users.
+
+#### When should I use the cache invalidate API?
```

Comment:
Is it intentional that these are H3 and H4 headings? Shouldn't all Qs on this page be H2 headings?

MDX source: 

<img width="492" alt="image" src="https://github.com/user-attachments/assets/69c823cc-af07-4645-b4ed-e3375d96218d">

Preview: 

<img width="538" alt="image" src="https://github.com/user-attachments/assets/75b205a9-695b-482c-b057-3084a55ce66e">

Reply:
Thanks for the quick review @nikolasburk. This was an oversight on my part. I resolved it.